src/test/playwright/support/pageobjects/course/CourseMessagesPage.ts

```diff
@@ -18,15 +18,16 @@ export class CourseMessagesPage {
      * Clicks the button to initiate channel creation.
      */
     async createChannelButton() {
-        await this.page.locator('#plusButton-generalChannels').click();
-        await this.page.locator('.modal-content #createChannel').click();
+        await this.page.click('.square-button > .ng-fa-icon');
```

Comment:
As I see it, all three buttons (create channel, browse channels, create group chat) are under the same menu and can be accessed via the same button.

Why do you use `this.page.click('.square-button > .ng-fa-icon');` here but `this.page.locator('.btn-primary.btn-sm.square-button').click()` in the other methods? Could we simply assign an ID to the button that opens the dropdown and use that to find it?